cowprotocol/explorer
# Adding Competition Solver tab to settlements page

# Summary

This pull request introduces a new feature to the settlements page - the "Competition Solver tab". This tab provides a detailed view of solver competitions, showing relevant data such as Auction ID, Auction Start Block, and more. The intention behind this tab is to give users a comprehensive overview of solver competitions, aiding in transparency and insight.

![image](https://github.com/cowprotocol/explorer/assets/30964272/7b3b3e80-fd36-4475-b0ce-967fbe7a3fe5)
# To Test

1. Access the Competition Solver tab:
* Open the settlements page ../tx/{txid} .
* Navigate to the new "Competition Solver tab".
* Expectation: The tab should display data entries like Auction ID, Auction Start Block, Liquidity Collected Block, etc.
* The data should be organized in rows with copy buttons, aiding easy sharing.
2. Interact with UI elements:
* Try the copy button to check if data gets copied to clipboard efficiently.
* Hover over the tooltips to see concise explanations.
 * Tooltips should provide a clear brief about each entry.
